src/main/java/space/obminyashka/items_exchange/controller/UserController.java

```diff
@@ -105,6 +105,21 @@ public Page<AdvertisementTitleDto> getFavoriteAdvertisements(
         return advService.findAllFavorite(authentication.getName(), PageRequest.of(page, size));
     }
 
+    @PreAuthorize("hasAnyRole('USER', 'ADMIN', 'MODERATOR')")
+    @DeleteMapping(value = ApiKey.USER_DELETE_MY_FAVORITE, produces = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(summary = "Delete user's favorite advertisement")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "202", description = "ACCEPTED"),
+            @ApiResponse(responseCode = "400", description = "BAD REQUEST"),
+            @ApiResponse(responseCode = "401", description = "UNAUTHORIZED")})
+    @ResponseStatus(HttpStatus.ACCEPTED)
+    public void deleteFavoriteAdvertisement(
+            @Parameter(hidden = true) Authentication authentication,
+            @Parameter(name = "advertisement_id", description = "Id of advertisement for deleting from favorite adv")
+            @PathVariable(name = "advertisement_id") UUID advertisementId) {
```

Comment:
Could you explain why the advertisement ID cannot be in the camel case in the path variable?